README.md

```diff
@@ -187,6 +187,19 @@ To query your asset use the following query:
 }
 ```
 
+If you do not want this plugin to download your media (images, videos, and file attachments), you can use the `skipFileDownloads` configuration option
```

Comment:
In the PR description you say:

>This may be useful for a site that is using an image CDN such as cloudinary: the CDN provides transformations, so we don't need to use Gatsby's transformation layer. 

I think it would be nice to add that here to, to make it more clear why users may want to use that option. And also mention that it would reduce build times